package.json

```diff
@@ -7,7 +7,7 @@
     "format:check": "prettier --check \"**/*.md\""
   },
   "devDependencies": {
-    "prettier": "^2.6.1"
+    "prettier": "2.6.2"
```

Comment:
**suggestion:** The [semantic versioning](https://docs.npmjs.com/about-semantic-versioning#using-semantic-versioning-to-specify-update-types-your-package-can-accept) `^` should cater for `2.x.x`. Running `npm update prettier` should update the `package-lock.json` with `2.6.2` and we can leave this unchanged.